### DISCUSSION TITLE
PDF Export Image

### DISCUSSION THREAD

**Discussion by IsThisOneAvailable365** [inkstitch/inkstitch#1029] [Q&A]
Hi all, I'm trying to export a design to pdf I get details but no image! Any thoughts, please. I'm using Mac 10.14.6

**Comment by jiriks74** [comment]
Hello, I'm new to using this project, but I'll try to help.
How are you exporting it? Via extension or the program menu? If you launch it from  terminal are any errors there?
I don't know how to debug this project, but this is how you can sometimes get some extra info.

**Reply by IsThisOneAvailable365** [reply]
Thanks for this but it is now sorted. I installed a beta version!
